.github/renovate.json5

```diff
@@ -64,31 +29,6 @@
       "groupName": "Spring Boot upgrades for v2",
       "description": "@akitsch: Spring Boot V3 requires Java 17"
     },
```

Comment:
**Thought:** I wonder if it's worth moving this to the global preset.
Point of Sale uses Java Spring Boot (see [related search in Point of Sale](https://github.com/search?q=repo%3AGoogleCloudPlatform%2Fpoint-of-sale%20spring&type=code)).

Reply:
This is a good callout, i'll add it to my doc.
Given that springboot 3 -> 4 is likely to also be disruptive, we probably want a way to say "no major updates for springboot".